libs/langchain/langchain/chat_models/mlflow.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from typing import List, Optional
+
+from langchain_core.pydantic_v1 import BaseModel, Extra
+
+from langchain.llms.databricks import ChatDatabricks
+
+logger = logging.getLogger(__name__)
+
+
+# Ignoring type because below is valid pydantic code
+# Unexpected keyword argument "extra" for "__init_subclass__" of "object"  [call-arg]
+class ChatParams(BaseModel, extra=Extra.allow):  # type: ignore[call-arg]
+    """Parameters for the `MLflow` LLM."""
+
+    temperature: float = 0.0
+    n: int = 1
+    stop: Optional[List[str]] = None
+    max_tokens: Optional[int] = None
+
+
+class ChatMlflow(ChatDatabricks):
```

Comment:
I was thinking the other way around: `ChatMlflow` contains the implementation and `ChatDatabricks` extends `ChatMlflow`.
It is strange to have the OSS MLflow branded API depend on Databricks-branded API. 
Also, since we are using `mlflow.deployments` under the hood, it is natural to put the implementation in `ChatMlflow`. WDYT? @dbczumar

Reply:
`ChatMlflow` depends on optional dependencies while `ChatDatabricks` doesn't.

Reply:
> It is strange to have the OSS MLflow branded API depend on Databricks-branded API.

btw do users care?

Reply:
> ChatMlflow depends on optional dependencies while ChatDatabricks doesn't.

This actually doesn't matter. Never mind.

Reply:
@freemso Made `ChatDatabricks` inherit `ChatMlflow`.